pkg/internal/server/grpc/common_test.go

```diff
@@ -45,64 +47,70 @@ import (
 type testHarness struct {
 	*runtime.TestHarness
 
-	SecurityToken string
-	DriverName    string
-	Audience      string
-	Namespace     string
-
-	fakeKubeClient     *fake.Clientset
-	fakeSnapshotClient *fakesnapshot.Clientset
-	fakeCBTClient      *fakecbt.Clientset
-
-	grpcServer *grpc.Server
-	listener   *bufconn.Listener
+	Audience                string
+	DriverName              string
+	Namespace               string
+	SecretName              string
+	SecretNs                string
+	SecurityToken           string
+	VolumeSnapshotClassName string
+
+	FakeKubeClient     *fake.Clientset
+	FakeSnapshotClient *fakesnapshot.Clientset
+	FakeCBTClient      *fakecbt.Clientset
+
+	lenFakeKubeClientReactionChain     int
+	lenFakeSnapshotClientReactionChain int
+	lenFakeCBTClientReactionChain      int
+
+	mockCSIDriverConn *grpc.ClientConn
+	grpcServer        *grpc.Server
+	listener          *bufconn.Listener
 }
 
 func newTestHarness() *testHarness {
 	return &testHarness{
-		SecurityToken: "securityToken",
-		DriverName:    "driver",
-		Audience:      "audience",
-		Namespace:     "namespace",
+		Audience:                "audience",
+		DriverName:              "driver",
+		Namespace:               "namespace",
+		SecretName:              "secret-name",
+		SecretNs:                "secret-ns",
+		SecurityToken:           "securityToken",
+		VolumeSnapshotClassName: "csi-snapshot-class",
 	}
 }
 
 func (th *testHarness) WithMockCSIDriver(t *testing.T) *testHarness {
 	th.TestHarness = runtime.NewTestHarness().WithMockCSIDriver(t)
+	th.mockCSIDriverConn = th.MockCSIDriverConn
```

Comment:
This makes `WithMockCSIDriver` use the same pattern as `WithFakeClientAPIs` w.r.t. `Runtime`.